docs/src/pages/docs/crocks/Writer.md

```diff
@@ -0,0 +1,352 @@
+---
+title: "Writer"
+description: "Writer Crock"
+layout: "guide"
+functions: ["log", "read"]
+weight: 160
+---
+
+
+```haskell
+Monoid m => Writer m a
+```
+
+The `Writer` monad makes it possible to represent and compose computations that produce some 
+additional data along with their computed values. `Writer` is a `Product` type that consists 
+of a `Monoid` as the first value that typically holds metadata, while the 
+second value holds the result of a computation. 
+
+<article id="topic-implements">
+
+## Implements
+`Functor`, `Apply`, `Chain`, `Applicative`, `Monad`
+
+</article>
+
+<article id="topic-construction">
+
+## Construction
+
+```haskell
+Writer :: MonoidTypeRep M, Monoid m => M -> (m, a) -> Writer m a
+```
+
+`Writer` is a type constructor that takes a `Monoid` constructor and returns a new constructor. The new constructor is a binary function that accepts a `Monoid` instance and a value `a`, and returns a `Writer` instance.
```

Comment:
The lines should be a max of 80 characters

Reply:
We want to break up those lines. Makes code review easier in a lot of cases.

Reply:
What do you think of https://eslint.org/docs/rules/max-len? It does the job pretty well, for files including markdown.

Reply:
P.S: I ran a lint check with this rule activated across the codebase, and it flags around 2500 cases. So I'm a bit lost here. IMHO, this open-to-interpretation nature of this requirement is not ideal.

Reply:
Really just care about the markdown in the docs/src/pages folder for these block descriptions.